dubbo-rpc/dubbo-rpc-api/src/main/java/org/apache/dubbo/rpc/filter/AccessLogFilter.java

```diff
@@ -75,6 +75,20 @@
 
     private static final long LOG_OUTPUT_INTERVAL = 5000;
 
+    static ThreadLocal<SimpleDateFormat> FILE_DATE_FORMATTER= new ThreadLocal<SimpleDateFormat>() {
+        @Override
+        protected SimpleDateFormat initialValue() {
+            return new SimpleDateFormat(FILE_DATE_FORMAT);
+        }
+    };
+
+    static ThreadLocal<SimpleDateFormat> MESSAGE_DATE_FORMATTER= new ThreadLocal<SimpleDateFormat>() {
```

Comment:
Hi, khan
We have integrated Netty's FastThreadLocal in dubbo. I think we can use it instead of ThreadLocal. I suggest you refer to RpcContext.LOCAL.

Reply:
Thanks for suggesting this. I was looking into RpcContext code for LOCAL and SERVER implementation, I would like to go this approach.